server/lib/front.go

```diff
@@ -142,6 +144,35 @@ func (sv *FrontServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
     c.reader(sv)
 }
 
+func (sv *FrontServer) initActors() error {
+	config := sv.config
+	//initialize actor system
+	if err := yue.Init(&yue.Config {
+		DatabaseAddress: config.DBHost,
+		CertPath: config.DBCertPath,
+		HostAddress: config.NodeIpv4Address,
+	}); err != nil {
+		return err
+	}
+	yue.Register("/hello", yue.ActorConfig {
+		SpawnConfig: yue.SpawnConfig {
+			Factory: yue.InmemoryExecuterFactory {
+				Constructor: actors.NewHelloActor,
+			},
+		},
+		Size: 1,
+	}, "channer")
+	go func() {
+		var r string
+		if err := yue.Call("/hello", "Hello", "actor-caller", &r); err != nil {
+			log.Fatalf("err should not happen: %v %v", err)
```

Comment:
__[go_vet]__
* [vet] missing argument for Fatalf("%v"): format reads arg 2, have only 1 args